From 4f4dcf8d828ae75d415c627aace50914a6b556e2 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 21 Jul 2020 10:45:55 +0800
Subject: [PATCH] drm/rockchip: use DRM_BUS_FLAG_PIXDATA_DRIVE_NEGEDGE

use DRM_BUS_FLAG_PIXDATA_DRIVE_NEGEDGE to instead of
DRM_BUS_FLAG_PIXDATA_NEGEDGE.

Fixes: d5272ec9bdbc ("drm/rockchip: use DRM_BUS_FLAG_PIXDATA_NEGEDGE to identify dclk polarity")
Change-Id: Ifaa376f1d71e77a83096c1e8fd8e1d245fea63d5
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_drm_vop.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_vop.c b/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
index ab88d1bc84aa..185e833e9269 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
@@ -2909,7 +2909,7 @@ static void vop_crtc_atomic_enable(struct drm_crtc *crtc,
 	if (vop->mcu_timing.mcu_pix_total)
 		vop_mcu_mode(crtc);
 
-	dclk_inv = (s->bus_flags & DRM_BUS_FLAG_PIXDATA_NEGEDGE) ? 1 : 0;
+	dclk_inv = (s->bus_flags & DRM_BUS_FLAG_PIXDATA_DRIVE_NEGEDGE) ? 1 : 0;
 
 	VOP_CTRL_SET(vop, dclk_pol, dclk_inv);
 	val = (adjusted_mode->flags & DRM_MODE_FLAG_NHSYNC) ?
-- 
2.35.3

